app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ideas for digital public works
```

Comment:
`works` or `goods`? I’m not sure on the definition of one versus the other but have seen digital public goods mentioned more.

Reply:
Oh, I’ve just read the first line 🤦